modin/pandas/test/test_io.py

```diff
@@ -915,6 +915,13 @@ def test_read_csv_categories(self):
             dtype={"one": "int64", "two": "category"},
         )
 
```

Comment:
Shouldn't we add a test for glob as well?

Reply:
We already have some glob tests for S3 that should test the new code paths but I can add one for gs as well.